charts/dynatrace-oneagent-operator/v0.6.0/crds/customresourcedefinition.yaml

```diff
@@ -0,0 +1,368 @@
+# Copyright 2019 Dynatrace LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
```

Comment:
you can move this CRD file to the templates folder and use `crd-install` hook to pre-install the CRDs so user doesn't need to apply this file manually.
``` 
annotations:
    "helm.sh/hook": "crd-install"
```